### PR TITLE
Remove ROS1 support

### PR DESCRIPTION
**Pull request by MNikoliCC**
**Description**
Remove ROS 1 support from Webots, including dependencies, code, and documentation references.

**Tasks**
Add the list of tasks of this PR.
  - [x] Remove ROS 1 support from Webots, including dependencies
  - [x] Update documentation to reflect the removal of ROS 1 support

**Additional context**
This PR aims to streamline the codebase by fully removing ROS 1 support. Note any related issues, deprecations, or compatibility changes here.

### CODE REVIEW

File: docs/reference/changelog-r2024.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Released on December **th, 2023.
     - Improved the speed and accuracy of box-plane collisions ([#6688](https://github.com/cyberbotics/webots/pull/6688)).
   - Cleanup
     - Removed deprecated `windowPosition`, `pixelSize` fields of [Display](display.md) node ([#6327](https://github.com/cyberbotics/webots/pull/6327)).
+    - Remove ROS1 support ([#6697](https://github.com/cyberbotics/webots/pull/6697)).
```
Comment (lukicdarkoo) [inline docs/reference/changelog-r2024.md:24]:
```suggestion
    - Removed ROS (1) support ([#6697](https://github.com/cyberbotics/webots/pull/6697)).
```

File: docs/reference/robot.md [hunk 2]
```diff
@@ -2377,4 +2244,4 @@ while ((message = wb_robot_wwi_receive_text())) {
 ```
 The reading head will be reset to the beginning of the buffer of received messages each time a time step is performed.
 
-> **note** [Java, Python, MATLAB, ROS]: `wb_robot_wwi_receive` and `wb_robot_window_send` functions are not available in the Java, Python, MATLAB, or ROS API.
+> **note** [Java, Python, MATLAB]: `wb_robot_wwi_receive` and `wb_robot_window_send` functions are not available in the Java, Python or MATLAB API.
```
Comment (lukicdarkoo) [inline docs/reference/robot.md:2247]:
```suggestion
> **note** [Java, Python, MATLAB]: `wb_robot_wwi_receive` and `wb_robot_window_send` functions are not available in the Java, Python, or MATLAB API.
```

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
Looks good, I believe there are only two places left:
- `.github/workflows/test_suite_linux_develop.yml`
- `.github/workflows/test_suite_linux_develop.yml`

**Comment by MNikoliCC** [comment]
> Looks good, I believe there are only two places left:
> 
> * `.github/workflows/test_suite_linux_develop.yml`
> * `.github/workflows/test_suite_linux_develop.yml`

Aren't those the same files?
The file is already modified.

**Comment by MNikoliCC** [comment]
> @MNikoliCC Sorry, the other file is `.github/workflows/test_suite_linux.yml`. It uses the non-existing flag `--exclude-ros`.
> 
> Regarding `.github/workflows/test_suite_linux_develop.yml`, no, this file is not changed, you changed only `test_suite_linux.yml`.

Sorry for the oversight, here are the changes.

**Review by lukicdarkoo — APPROVED** [review]
Thank you

**Pull request merged**